examples/deployment_crud.py

```diff
@@ -13,8 +13,7 @@
 # limitations under the License.
 
 """
-This example shows how to work with AppsV1Api to create, modify and delete
-deployments
+Creates, updates, and deletes a deployment.
```

Comment:
Talking about AppsV1API is important since we also can do this with ExtensionsAPI.

Reply:
Fixed this, will push on the next commit after more review comments.